tests/consensus_tests/auth_tests/test_jwt_validation.py

```
@@ -59,6 +59,11 @@ def scroll_with_token(collection: str, token: str) -> requests.Response:
 
 
 def test_value_exists_claim():
+    value_exists_claim_check(SECRET)
+    value_exists_claim_check(ALT_SECRET)
+
+s
```

Ruff (0.14.10)

Found useless expression. Either assign it to a variable or remove it.

(B018)